test/test_yum_plugins/test_enabled_repos_upload.py

```diff
@@ -45,7 +45,7 @@ def test_send(self, cache_save, cache_valid, fake_report_enabled, fake_read, fak
     @patch('katello.repos.report_enabled_repos')
     @patch('katello.repos.EnabledRepoCache.is_valid')
     @patch('katello.repos.EnabledRepoCache.save')
-    @unittest.skipIf(sys.version_info[0] > 2, "yum tests for PY2 only")
```

Comment:
these two skipIf's need to be added back. they started failing when I remove the DNF stuff but I don't actually have an explanation as to why!